### PR TITLE
Translate babel error message and source code comments

### PR DESCRIPTION
**Pull request by abnersajr**
This PR references issue #1 translating the `CodeEditor` component.

### CODE REVIEW

**Comment by fjoshuajr** [comment]
Working fast @abnersajr 👍

File: src/components/CodeEditor/CodeEditor.js [hunk 1]
```diff
@@ -280,8 +278,8 @@ class CodeEditor extends Component {
     } catch (error) {
       console.error(error);
 
-      // Certain ad blockers (eg Fair AdBlocker) prevent Babel from loading.
-      // If we suspect this is the case, we can show a more helpful error.
+      // Certos Bloqueadores de Anúncios (eg Fair AdBlocker) evitam que o Babel carregue.
```
Comment (fjoshuajr) [inline src/components/CodeEditor/CodeEditor.js]:
```suggestion
      // Certos bloqueadores de anúncios (ex: Fair AdBlocker) evitam que o Babel carregue.
```

File: src/components/CodeEditor/CodeEditor.js [hunk 2]
```diff
@@ -56,12 +56,10 @@ class CodeEditor extends Component {
     if (showBabelErrorMessage) {
       errorMessage = (
         <span>
-          Babel could not be loaded.
+          O Babel não pode ser carregado.
```
Comment (fjoshuajr) [inline src/components/CodeEditor/CodeEditor.js]:
```suggestion
          O Babel não pôde ser carregado.
```

File: src/components/CodeEditor/CodeEditor.js [hunk 3]
```diff
@@ -56,12 +56,10 @@ class CodeEditor extends Component {
...
           <br />
           <br />
-          This can be caused by an ad blocker. If you're using one, consider
-          adding reactjs.org to the whitelist so the live code examples will
-          work.
+          Isto pode ser causado por algum bloqueador de anúncio. Se você está usando um, considere adicionar o endereço reactjs.org na <i>whitelist</i>, ou lista de permitidos, e feito isso os exemplos com <i>live code</i> irão funcionar.
```
Comment (fjoshuajr) [inline src/components/CodeEditor/CodeEditor.js]:
```suggestion
          Isto pode ser causado por algum bloqueador de anúncio. Se você está usando algum, considere adicionar o endereço reactjs.org na <i>whitelist</i>, ou lista de permitidos, e feito isso os exemplos com <i>live code</i> irão funcionar.
```

Comment (fjoshuajr) [reply src/components/CodeEditor/CodeEditor.js]: I'm not sure whether we should use italics for non-Portuguese words or we should keep plain text.

cc @glaucia86

Comment (fjoshuajr) [reply src/components/CodeEditor/CodeEditor.js]: https://github.com/reactjs/pt-BR.reactjs.org/issues/4#issuecomment-461955347

**Comment by abnersajr** [comment]
I've accepted all the requests. Thank you @fjoshuajr for those catches.

File: src/components/CodeEditor/CodeEditor.js [hunk 4]
```diff
@@ -56,12 +56,10 @@ class CodeEditor extends Component {
...
       errorMessage = (
         <span>
-          Babel could not be loaded.
+          O Babel não pôde ser carregado.
           <br />
           <br />
-          This can be caused by an ad blocker. If you're using one, consider
-          adding reactjs.org to the whitelist so the live code examples will
-          work.
+          Isto pode ser causado por algum bloqueador de anúncio. Se você está usando algum, considere adicionar o endereço reactjs.org na <i>whitelist</i>, ou lista de permitidos, e feito isso os exemplos com <i>live code</i> irão funcionar.
```
Comment (cezaraugusto) [inline src/components/CodeEditor/CodeEditor.js]: adblock FTW

**Pull request merged**